0xProject/0x-monorepo
# [website] Prepare website for breaking MetaMask change for EIP 1102

## Description

* Update our injected web3 handling to now handle injected provider instead, and request user address information using the new `provider.enable()` method 
* Learn about MetaMask privacy mode here: https://medium.com/metamask/https-medium-com-metamask-breaking-change-injecting-web3-7722797916a8

## Testing instructions

* Download custom build of MetaMask and install as the medium article above describes
* Enable privacy mode
* Visit http://staging-0xproject.s3-website-us-east-1.amazonaws.com/portal
* Observe MetaMask asking for approval

## Types of changes





* New feature (non-breaking change which adds functionality)



## Checklist:
